draft-ietf-httpbis-resumable-upload.md

```diff
@@ -406,6 +409,10 @@ The `Upload-Complete` request and response header field indicates whether the co
 
 The `Upload-Complete` header field MUST only be used if support by the resource is known to the client ({{feature-detection}}).
 
+# Media Type `application/partial-upload`
+
+The `application/partial-upload` media type describes a contiguous block of data that should be uploaded to a resource. There is no minimum size on the block of data and it might be empty. The start and end of block of data might align with the start and end of the file that should be uploaded, but are not required to do so.
+
```

Comment:
Minor nit: 

> The start and end of *THE* block of data might align with the start and end of the file that should be uploaded, but are not required to do so.

or 

> The start and end of *THE* data might align with the start and end of the file that should be uploaded, but are not required to do so.


Should we avoid using "file" as the thing to be uploaded and use something more generic like "resource that should be uploaded"?